src/locales/de.json

```
@@ -107,6 +107,17 @@
             "UpdatingDone": "Aktualisierung {software} abgeschlossen!"
         }
     },
+    "BedScrews": {
+        "Abort": "abbrechen",
+        "Adjusted": "angepasst",
+        "Accept": "bestätigen",
+        "Description": "Klick auf <b>ANGEPASST</b> wenn die aktuelle schraube angepasst wurde. Klick auf <b>BESTÄTIGEN</b> um weiterzufahren ohne Anpassung.",
+        "Headline": "Bettschrauben",
+        "ScrewAccepted": "bestätigte Schrauben",
+        "ScrewIndex": "Index der Schraube",
+        "ScrewName": "Name der Schrauben",
```

```suggestion
        "ScrewName": "Name der Schraube",
```
Is it a single screw name or multiple?